.github/workflows/build-development.yml

```diff
@@ -9,76 +9,76 @@ jobs:
-        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-        key: 1-${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          1-${{ runner.os }}-yarn-
-    - uses: actions/cache@v2
-      id: build-cache
-      with:
-        path: node_modules/.cache
-        key: 1-${{ runner.os }}-build-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          1-${{ runner.os }}-build-
-    - name: yarn install and build
-      run: |
-        yarn install
-        yarn run build
-      env:
-        REACT_APP_URL_SHORTENER_SIGNATURE: ${{ secrets.REACT_APP_URL_SHORTENER_SIGNATURE }}
-        REACT_APP_URL_SHORTENER_BASE: ${{ secrets.REACT_APP_URL_SHORTENER_BASE }}
-        REACT_APP_SENTRY_DSN: ${{ secrets.REACT_APP_SENTRY_DSN }}
-        REACT_APP_GOOGLE_CLIENT_ID: ${{ secrets.REACT_APP_GOOGLE_CLIENT_ID }}
-        REACT_APP_GOOGLE_API_KEY: ${{ secrets.REACT_APP_GOOGLE_API_KEY }}
-        REACT_APP_GOOGLE_APP_ID: ${{ secrets.REACT_APP_GOOGLE_APP_ID }}
-        REACT_APP_PLAYGROUND_ONLY: "TRUE"
-        REACT_APP_ENABLE_GITHUB_ASSESSMENTS: "TRUE"
-        REACT_APP_VERSION: ${{ format('{0}-{1}', github.sha, steps.get-time.outputs.time) }}
-        REACT_APP_ENVIRONMENT: "pages"
-        REACT_APP_MODULE_BACKEND_URL: https://source-academy.github.io/modules
-        REACT_APP_SHAREDB_BACKEND_URL: ${{ secrets.REACT_APP_SHAREDB_BACKEND_URL }}
-        PUBLIC_URL: "https://sourceacademy.org"
-        REACT_APP_GITHUB_OAUTH_PROXY_URL: ${{ secrets.REACT_APP_GITHUB_OAUTH_PROXY_URL }}
-        REACT_APP_GITHUB_CLIENT_ID: ${{ secrets.REACT_APP_GITHUB_CLIENT_ID }}
-    - name: Create Sentry release
-      working-directory: build
-      run: |
-        SENTRY_RELEASE="cadet-frontend@$REACT_APP_VERSION"
-        "$GITHUB_WORKSPACE/sentry-cli" releases new -p "$SENTRY_PROJECT" "$SENTRY_RELEASE"
-        "$GITHUB_WORKSPACE/sentry-cli" releases set-commits --auto "$SENTRY_RELEASE"
-        "$GITHUB_WORKSPACE/sentry-cli" releases files "$SENTRY_RELEASE" upload-sourcemaps --url-prefix '~/static/js' --rewrite static/js
-        "$GITHUB_WORKSPACE/sentry-cli" releases finalize "$SENTRY_RELEASE"
-        "$GITHUB_WORKSPACE/sentry-cli" releases deploys "$SENTRY_RELEASE" new -e pages
-      env:
-        SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-        SENTRY_ORG: sourceacademy
-        SENTRY_PROJECT: cadet-frontend
-        REACT_APP_VERSION: ${{ format('{0}-{1}', github.sha, steps.get-time.outputs.time) }}
-    - name: Remove sourcemaps
-      working-directory: build
-      run: |
-        find -name '*.map' -print -delete
-    - name: Deploy
-      uses: peaceiris/actions-gh-pages@v3
-      with:
-        external_repository: source-academy/sourceacademy.org
-        deploy_key: ${{ secrets.DEPLOY_PRIVATE_KEY }}
-        publish_dir: ./build
-        publish_branch: master
-        force_orphan: true
+      - uses: actions/checkout@master
+      - name: Use Node.js
+        uses: actions/setup-node@v2-beta
```

Comment:
Can we standardise and use `actions/setup-node@v3` instead?

Reply:
Oh, this commit was just for the bumping and my IDE auto-formatted the YAML. I've already updated it to v3 in a subsequent commit.

I forgot that v4 has already came out haha; I don't see any reason why we should stick to v3 so I'll bump it, although I do want to keep the `node-version: 20` in the YAML, just to lock it and also make it clear to developers what version we're using.